src/ui/ResponsiveComponents/ResponsiveDropdown/ResponsiveDropdownModalContent.ts

```diff
@@ -0,0 +1,55 @@
+import { IResponsiveDropdownContent, ResponsiveDropdownContent } from './ResponsiveDropdownContent';
+import { Dom, $$, l } from '../../../Core';
+import { SVGIcons } from '../../../utils/SVGIcons';
+import { FocusTrap } from '../../FocusTrap/FocusTrap';
+
+export class ResponsiveDropdownModalContent implements IResponsiveDropdownContent {
+  private closeButton: Dom;
+  private className: string;
+  private focusTrap: FocusTrap;
+
+  constructor(private componentName: string, public element: Dom, private closeButtonLabel: string, private close: () => void) {
+    this.className = `coveo-${this.componentName}-dropdown-modal-content`;
+  }
+
+  private set hidden(shouldHide: boolean) {
+    this.element.toggleClass('coveo-hidden', shouldHide);
+  }
+
+  public positionDropdown(): void {
+    this.element.el.classList.add(this.className, ResponsiveDropdownContent.DEFAULT_CSS_CLASS_NAME);
+    this.element.setAttribute('role', 'group');
+    this.element.setAttribute('aria-label', l('FiltersDropdown'));
+    this.hidden = false;
+    this.closeButton = $$(
+      'button',
+      {
+        className: 'coveo-facet-modal-close-button',
+        ariaLabel: this.closeButtonLabel
+      },
+      SVGIcons.icons.mainClear
```

Comment:
The icon won't display in all browsers this way (Chrome). I think you need to set the height/width in css to the SVG directly, or put the SVG in a div and set height/width to that div.

```
$$('div', { className: 'some-clear-icon' }, SVGIcons.icons.mainClear);
```
Then in sass
```
.coveo-facet-modal-close-button {
        $close-button-size: 32px;
        $close-button-icon-size: 12px;
        .some-clear-icon {
          width: $close-button-size;
          height: $close-button-size;
        }
.....
}
```

Reply:
Alright, I'll set the size of the SVG directly.